src/components/ToDoItem.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
This file looks very similar to `CompleteItem.jsx`

do you think you could re-use the component for items both completed and todo?